codecov.yml

```
@@ -0,0 +1,22 @@
+coverage:
+  status:
+    patch:
+      default:
+        target: 85%
+        informational: true
+    project:
+      default:
+        target: 65%
+        threshold: 1%
+        informational: true
```

```suggestion
      default:
        informational: true
    project:
      default:
        informational: true
```

I don't really want to argue about targets right now, so would prefer we start with just printing the status.

---

OK, let's start by simply printing the status for now.